lib/calendar_translations/translations.ex

```diff
@@ -535,6 +535,112 @@ defmodule CalendarTranslations.Translations do
   def date_format(:"zh-TW"), do: {:ok, "%Y-%m-%d"}
   def date_format(:"zh-YUE"), do: {:ok, "%Y-%m-%d"}
 
+  def time_format(:af), do: {:ok, "%H:%M"}
+  def time_format(:ar), do: {:ok, "%H:%M"}
+  def time_format(:az), do: {:ok, "%H:%M"}
+  def time_format(:be), do: {:ok, "%H:%M"}
+  def time_format(:bg), do: {:ok, "%H:%M"}
+  def time_format(:bn), do: {:ok, "%H:%M"}
+  def time_format(:bs), do: {:ok, "%H:%M:%S"}
+  def time_format(:ca), do: {:ok, "%H:%M"}
+  def time_format(:cs), do: {:ok, "%H:%M"}
+  def time_format(:csb), do: {:ok, "%H:%M"}
+  def time_format(:cy), do: {:ok, "%H:%M"}
+  def time_format(:da), do: {:ok, "%H.%M"}
+  def time_format(:de), do: {:ok, "%H:%M Uhr"}
```

Comment:
I am German and here you can write timestamps both with and without the "Uhr" suffix. I am honestly not so sure whether we can make this choice for the user. I'd rather be as consistent as possible across locales by choosing the simplest possible version. In this case it would make sense for me to drop the suffix. Not sure whether this is optional in other languages as well.